### PR TITLE
Split microservices into separate packages

### LINKED ISSUE #10159: Split reqmgr2ms into separate PyPI packages
**Impact of the new feature**
Affects microservices currently packaged as the "reqmgr2ms" PyPI package and RPM

**Is your feature request related to a problem? Please describe.**
Each microservice runs in its own container and therefore should have its own PyPI package.

**Describe the solution you'd like**
Create PyPI packages for each specific reqmgr2 microservice, instead of one large reqmgr2ms package.

- MSTransferor
- MSMonitor
- MSOutput
- MSRuleCleaner
- MSUnmerged

**Describe alternatives you've considered**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by goughes**
Fixes #10159

#### Status
Not tested

#### Description
This PR updates `build_dependencies.py`, creates a new reqmgr2ms-core system and adds separate packages for all the current microservices. 

#### Is it backward compatible (if not, which system it affects?)
Yes, we leave the current `reqmgr2ms` package which is still used for RPM based builds.

#### Related PRs
Follow up to https://github.com/dmwm/WMCore/pull/10357 as I have created a "MSCore" directory and moved MSCore, MSManager and TaskManager modules to it.

#### External dependencies / deployment changes
We will need to update each microservice's config.py since the path for MSManager has changed. 
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/168
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/167
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/166
https://github.com/dmwm/deployment/pull/1192

### CODE REVIEW

File: tools/build_pypi_packages.sh [hunk 1]
```diff
@@ -14,7 +14,9 @@ set -x
 # package passed as parameter, can be one of PACKAGES or "all"
 TOBUILD=$1
 # list of packages that can be built and uploaded to pypi
-PACKAGES="wmagent wmagent-devtools wmcore reqmon reqmgr2 reqmgr2ms global-workqueue acdcserver"
+PACKAGES="wmagent wmagent-devtools wmcore reqmon reqmgr2 reqmgr2ms global-workqueue acdcserver
```
Comment (amaltaro) [inline tools/build_pypi_packages.sh:17]: Perhaps we should stop building/uploading `reqmgr2ms` to PyPi?

**Review by amaltaro — COMMENTED** [review]
Thanks Erik, changes look good to me. I left one comment along the code though.
I think we should stop building `reqmgr2ms` for PyPi, even though we still need it for RPMs.

**Comment by goughes** [comment]
Thanks @amaltaro, I have removed reqmgr2ms from requirements.txt and the PyPI build script.

**Review by amaltaro — APPROVED** [review]
Thanks Erik, this looks good to go!

**Comment by amaltaro** [comment]
Thanks Erik!

**Pull request merged**